frontend/src/bundles/cv-editor/components/user-flow-bar/types/interface.ts

```diff
@@ -0,0 +1,12 @@
+type Map = Record<string, string | undefined>;
+  
+const UserFlowBtns: Map = {
```

Comment:
Try to find better locations for this `const`. It is should be in `types/interface.ts`?